### ISSUE TITLE
Fix multiple issues with the wireless information reported on FreeBSD

### ISSUE THREAD

**Issue by bsdjhb** [i3/i3status#57]
Again, sorry I do not have a pull request already done.  However, here is a link to the commit to a patch in the FreeBSD ports tree along with a description of the fixes:

https://svnweb.freebsd.org/ports?view=revision&revision=395523

Fix multiple issues with the wireless information reported by i3status for
an interface:

- Instead of hoping that the currently associated AP will show up as
  the first AP in the list of scan results, fetch the BSSID of the
  currently associated AP and use that to fetch station info for the
  AP.  This provides more frequently updated signal strength
  information than scan results and reliably provides information when
  multiple APs are in range.
- Do not treat the RSSI value as a raw signal value in dBm.  Instead, use
  the same formula as ifconfig(8) to compute a signal value.
- Do not report the beacon interval value as a signal quality level.

**Comment by stapelberg** [comment]
As with #56, can you please send this as a pull request?

**Comment by bsdjhb** [comment]
Done as #59   Once this is merged I can implement the frequency info on FreeBSD as a followup.

**Comment by stapelberg** [comment]
Thanks!
